docs/csharp/methods.md

```diff
@@ -136,8 +136,11 @@ A method definition can specify that its parameters are required or that they ar
 The parameter's default value must be assigned by one of the following kinds of expressions:
 
 - A constant, such as a literal string or number.
+- An expression of the form `default(SomeType)`, where `SomeType` can be either a value type or a reference type. If it's a reference type, it's effectively the same as specifying `null`. Beginning with C# 7.1, you can use the `default` literal, as the compiler can infer the type from the parameter's declaration.
 - An expression of the form `new ValType()`, where `ValType` is a value type. Note that this invokes the value type's implicit parameterless constructor, which is not an actual member of the type.
-- An expression of the form `default(SomeType)`, where `SomeType` can be either a value type or a reference type. If it's a reference type, it's effectively the same as specifying `null`.
+
+  > [!NOTE]
+  > In C# 10.0 and later, an expression of the form `new ValType()` is not allowed if it invokes the explicitly defined parameterless constructor of a value type. Use the `default(ValType)` expression or the `default` literal to provide the parameter's default value. For more information about parameterless constructors, see the [Parameterless constructors and field initializers](language-reference/builtin-types/struct.md#parameterless-constructors-and-field-initializers) section of the [Structure types](language-reference/builtin-types/struct.md) article.
```

Comment:
This addresses https://github.com/dotnet/docs/pull/24234#discussion_r632409776

Reply:
I'm not sure what you mean by the first sentence in this paragraph. You can invoke the parameterless constructor. I don't have a suggestion, because I'm not sure what your purpose is.

Reply:
I mean that the following code generates the error `Error CS1736 Default parameter value for 'entry' must be a compile-time constant`:
```csharp
    public static void Display(Example entry = new Example())
    {    }

    public struct Example
    {
        public Example() { }
    }
```

The error disappears as soon as I remove the explicit parameterless constructor.

In that sense, "is not allowed" equivalent to "results in the compiler error"

Reply:
@BillWagner I've rephrased the sentence, please check it again.

Reply:
I like the way you rephrased this. It's more clear when the restriction applies, and why.